packages/util/test/types.spec.ts

```diff
@@ -137,35 +135,3 @@ tape('toType', function (t) {
     })
   })
 })
-
-tape('isFalsy and isTruthy', function (t) {
```

Comment:
If we delete these tests, then the `isTruthy`, `isFalsy` methods should also be removed from `util`.

Reply:
Oh indeed, thought I had removed, forgot to commit